connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerTest.java

```diff
@@ -1144,6 +1144,10 @@ public void testAdminConfigsClientOverridesWithAllPolicy() {
         connConfig.put("metadata.max.age.ms", "10000");
 
         Map<String, String> expectedConfigs = new HashMap<>();
+        expectedConfigs.putAll(props);
+        expectedConfigs.remove("admin.client.id");
+        expectedConfigs.remove("admin.metadata.max.age.ms");
```

Comment:
nit: these 2 are added by the testcase just a few lines above. You can construct expectedConfigs from workerProps directly.